baseplate/lib/message_queue.py

```diff
@@ -111,14 +147,100 @@ def unlink(self) -> None:
         self.queue.unlink()
 
     def close(self) -> None:
-        """Close the queue, freeing related resources.
+        self.queue.close()
 
-        This must be called explicitly if queues are created/destroyed on the
-        fly. It is not automatically called when the object is reclaimed by
-        Python.
 
-        """
-        self.queue.close()
+class InMemoryMessageQueue(MessageQueue):
+    """An in-memory inter process message queue.
+
+    Uses a simple Python Queue to store data.
+
+    Used in conjunction with the RemoteMessageQueue to
+    provide an alternative to Posix queues, for systems
+    that don't have Posix available. The client will
```

Comment:
This isn't really an accurate statement.  All hosts we run on will have POSIX queues available.  It's being able to configure the size of them that's unavailable.

Reply:
thank you, that's a helpful clarification. I updated the comment to be more accurate